sktime/transformers/series_as_features/signature_based/_rescaling.py

```diff
@@ -0,0 +1,124 @@
+"""
+rescaling.py
+=========================
+Signature rescaling methods. This implements the pre- and post- signature
+rescaling methods along with generic scaling methods along feature
+dimensions of 3D tensors.
+
+Code for `rescale_path` and `rescale_signature` written by Patrick Kidger.
+"""
+import math
+import torch
+from sklearn.preprocessing import StandardScaler, MinMaxScaler, MaxAbsScaler, \
+    FunctionTransformer
+from sktime.transformers.series_as_features.base import \
+    BaseSeriesAsFeaturesTransformer
+import signatory
+
+
+class TrickScaler(BaseSeriesAsFeaturesTransformer):
```

Comment:
same here, I'd suggest making this a private class, unless it's meant to be used directly by users, otherwise perhaps a somewhat more descriptive name